egs/heroico/s5/path.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
for path.sh, I'd suggest using the standard path.sh such as the one that can be found in the ws recipe + add modifications you need.